sheetstorm/settings/base.py

```diff
@@ -44,6 +44,7 @@
     'django_countries',
     'users',
     # 3rd party
+    'behave_django',
```

Comment:
I don't really like that this is added in base django settings. I would move it to `testing.py`

Reply:
Done